ci/vars-logging-test.yml

```diff
@@ -1,8 +1,12 @@
 ---
-post_deploy_00_fvt_logging_computes:
-  source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/logging_tests_computes.yml"
+# Skip running test_operator tests
+cifmw_test_operator_dry_run: true
```

Comment:
I don't think that we need this any longer. I _think_ that I added it to skip running the tempest tests when the tests were failing at some point.

Reply:
@mgirgisf based on Emma's comment. Shouldn't these 2 lines be deleted?